src/components/Dashboard/Header/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { ReactComponent as NimbleLogoWhite } from 'assets/images/icons/nimble-logo-white.svg';
+
+interface DashboardHeaderProps {
+  dateTime: string;
+  daysAgo: string;
+  profileUrl: string;
+  children: React.ReactNode;
+  'data-test-id'?: string;
+}
+const DashboardHeader = ({ dateTime, daysAgo, profileUrl, children, ...attributes }: DashboardHeaderProps): JSX.Element => {
+  return (
+    <div className="flex flex-col min-h-screen" {...attributes}>
+      <div className="flex flex-row justify-between pt-8">
+        <NimbleLogoWhite />
```

Comment:
The header is often an actionable link to return home, should we add a parent div for it?
And I think we might need to decrease the size a bit on the mobile screen 🤔

Reply:
@hoangmirs currently there is no design for mobile app and there is no requirements for handle responsive UI so please ignore it 😄 

> The header is often an actionable link to return home, should we add a parent div for it?

This logo is only appeared in home so I think we don't need to handle this action 🙏